app/assets/stylesheets/common.scss

```diff
@@ -750,6 +750,11 @@ tr.turn:hover {
 
   &.new-user-main {
     background-image: image-url("sign-up-illustration.png");
+    background-position-x: 100px;
```

Comment:
I would probably make this more like 40-50 pixels - it stops the image being cropped on the right and also means more of it appears with a smallish phone screen.

Reply:
I will reduce it to 50.

It is a double edged sword though, since if the tab labels are long words, the "Sign up" link colour would be difficult to read with blue hand of the globe.